tom.cf

```diff
@@ -0,0 +1,136 @@
+# Policy to update and run the Tom GitHub bot
+# Note: This is example policy, for production changes, go to private infra repo
+
+body file control
+{
+    inputs => { "$(sys.libdir)/stdlib.cf", "$(sys.libdir)/vcs.cf" };
```

Comment:
I changed this to $(sys.local_libdir) to fix up the duplicate definition of bundle issues I had